makePoplogCommander.sh

```diff
@@ -257,7 +278,7 @@ int howManyTimes( const char * haystack, const char * needle ) {
     return count;
 }
 
-void setEnvReplacingUSEPOP( char * name, char * value, char * base ) {
+void setEnvReplacingUSEPOP( char * name, char * value, char * base, int flags ) {
```

Comment:
Does `setEnvReplacingUSEPOP` really need access to all `flags`? Can't the `(flags && INHERIT_ENV) == 0` be pushed up the stack?

Reply:
Ta for commenting, Will. Yes good point. Of course it only needs the `overwrite` flag. I will fix.

Reply:
Done